Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -636,6 +637,26 @@ namespace AssetProcessor
             return;
         }
 
+        if (!m_lfsPointerFileValidator)
+        {
+            m_lfsPointerFileValidator = AZStd::make_unique<LfsPointerFileValidator>();
+        }
+
+        QString absolutePathToFile = jobEntry.GetAbsoluteSourcePath();
+
+        // Write the LFS pointer file related messages to the job log and display it in the job event details UI.
+        AssetProcessor::SetThreadLocalJobId(jobEntry.m_jobRunKey);
+        AssetUtilities::JobLogTraceListener jobLogTraceListener(jobEntry);
+
+        AZ_Error(AssetProcessor::ConsoleChannel, !m_lfsPointerFileValidator->IsLfsPointerFile(absolutePathToFile.toUtf8().constData()),
```

Comment:
Don't do actual work inside AZ_Error etc as its compiled out in release, unless you know your code never runs in release.